doc/scripts/GMT_App_M_1d.sh

```diff
@@ -5,16 +5,17 @@
 #
 # We have four sets of CPT figures to make:
 # 1a) Our regular, traditional GMT CPTs [44]
-# 1b) The regular Scientific Color Maps* [24]
+# 1b) The regular Scientific Color Maps* [30]
 # 1c) Categorical CPTs (ours and SCM*)  [18]
-# 1d) Cyclic CPTs form SCM* [4]
+# 1d) Cyclic CPTs from SCM* [5]
 #
 # *from Fabio [www.fabiocrameri.ch/visualisation]
 
 GMT_SHAREDIR=$(gmt --show-sharedir)
 
 # Here we list all the cyclic cpts from the SCM:
 cat << EOF > tt.lis
+bam0
```

Comment:
I think there may be a bug with this `doc/scripts/GMT_App_M_1d.sh` script because there's an odd number (5) of CPTs instead of even (4). If I take out any other CPT here (e.g. vikO) so that there is only 4 CPTs, then `ctest -V -R GMT_App_M_1d` works. But if I keep it at 5 CPTs, then it produces this error:

```
(pygmt) user@computer:~/Documents/github/gmt/build$ ctest -V -R GMT_App_M_1d
UpdateCTestConfiguration  from :/home/user/Documents/github/gmt/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/user/Documents/github/gmt/build/DartConfiguration.tcl
Test project /home/user/Documents/github/gmt/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 35
    Start 35: doc/scripts/GMT_App_M_1d.sh

35: Test command: /bin/bash "/home/user/Documents/github/gmt/build/test/gmtest" "doc/scripts/GMT_App_M_1d.sh"
35: Test timeout computed to be: 10000000
35: Set GMT_SESSION_NAME = 188583
35: makecpt [ERROR]: Options -C: No CPT argument given
35: ERROR: /home/user/Documents/github/gmt/build/test/gmtest:70
35: memtrack errors: 0
35: exit status: 1
1/1 Test #35: doc/scripts/GMT_App_M_1d.sh ......***Failed   12.79 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =  12.98 sec

The following tests FAILED:
	 35 - doc/scripts/GMT_App_M_1d.sh (Failed)
Errors while running CTest
```